go/enclave/genesis/genesis.go

```diff
@@ -45,17 +45,16 @@ func (g Genesis) CommitGenesisState(storage storage.Storage) error {
 	if err != nil {
 		return err
 	}
-	_, err = stateDB.Commit(0, false, true)
+	root, err := stateDB.Commit(0, false, true)
 	if err != nil {
 		return err
 	}
 
-	// todo - VERKLE
```

Comment:
remind me what this todo is for? Any impact of not using verkle here for now?

Reply:
on my last attempt to implement the verkle stuff, I left this comment over the sections that needed to be uncommented on the next retry